docs/providers/resources.md

```diff
@@ -1,10 +1,12 @@
 # Resource
+
```

Comment:
Let's not add new line after headers, please

Reply:
https://github.com/DavidAnson/markdownlint/blob/v0.35.0/doc/md022.md

Reply:
it's easier to read without these new lines. Text becomes more compact and it's easier to group some parts of content